### ISSUE TITLE
Changes to options not reflected

### ISSUE THREAD

**Issue by deejbee** [apertureless/vue-chartjs#365]
### Expected Behavior
When I change the options after initial render, specifically the xAxis time unit it should update the graph.

### Actual Behavior
Graph still uses previous axis config

Are changes to the options expected to update the graph or am I not doing something right?

I'm using a a wrapped line chart like this:

`import { Line, mixins } from 'vue-chartjs'
const { reactiveProp } = mixins

export default {
  extends: Line,
  mixins: [reactiveProp],
  props: ['options'],
  mounted () {
    // this.chartData is created in the mixin.
    // If you want to pass options please create a local options object
    this.renderChart(this.chartData, this.options)
  }
}`

### Environment
- vue.js version: 2.5.6
- vue-chart.js version: 3.3.1
- npm version: 6.1.0

**Comment by apertureless** [comment]
Hello @deejbee 

If you change the options "on-the-fly" you have to re-render the chart by yourself.

The reactiveMixin is only for the chart data. 

You can add a watcher to the options and then simply call `this.renderChart()` to re-render it

**Comment by deejbee** [comment]
I tried the watch as described here but when I modified the options it did not fire:

https://github.com/apertureless/vue-chartjs/issues/195

**Comment by aaronpeterson** [comment]
Watchers on props don't work at all for me.  Is something intercepting them?

**Comment by beatgrabe** [comment]
@apertureless I followed your approach, because I had a similar issue. It seems after rendering the chart with altered options again, the 'old' chart remains on the canvas. This is introducing further issues…

So I managed to get around the issue by destroying the old instance by calling `chart.destroy()` by myself. So for sake of completeness and helping others coming by, here is what I did:

```
watch: {
  'chartOptions': {
    handler: function (newChartOptions) {
      this.$data._chart.destroy()
      this.renderChart(this.chartData, newChartOptions)
    }
  }
}
```

Cheers!
